demos/privacy_preservation/README.md

```diff
@@ -0,0 +1,19 @@
+# Privacy Preservation Demo
+
+## Models
+
+The demo uses two models, one for object detection and one for pose estimation. You can download them here:
+- https://tfhub.dev/tensorflow/lite-model/efficientdet/lite2/detection/default/1
+- https://tfhub.dev/google/lite-model/movenet/singlepose/lightning/3
+
+## How to run
+`OMP_NUM_THREADS=4 AIO_PROCESS_MODE=0 python3 run.py --model-path lite-model_movenet_singlepose_lightning_3.tflite --detection-model-path lite-model_efficientdet_lite2_detection_default_1.tflite`
+
```

Comment:
Consider adding a requirements.txt file required to make this thing work in our release docker - that will actually mean just one dep:
flask

Reply:
![Screenshot 2022-09-09 at 14 11 35](https://user-images.githubusercontent.com/52271793/189347326-2975583c-59fe-4734-bae8-16df3aa65740.png)
Perhaps having some default welcome image would be nice - like big ampere logo or something

Reply:
Your write-up on this demo could be linked somewhere in this read-me.